### DISCUSSION TITLE
re: Use old proto version until maven issue is fixed

### DISCUSSION THREAD

**Discussion by dio** [google/j2cl#176] [General]
Sorry, just curious about the following TODO:

https://github.com/google/j2cl/blob/55cf0ed9de14142361fa4e0512ad4ac0a1b286ce/build_defs/repository.bzl#L6

Is this related to the following error: 

```
external/com_google_protobuf/java/util/BUILD.bazel:8:13: no such package '@maven//': The repository '@maven' could not be resolved: Repository '@maven' is not defined and referenced by '@com_google_protobuf//java/util:util'
```

when using `"@com_google_protobuf//:protobuf_java_util"` as one of the deps? Do you mind pointing out the upstream issue on this? 

Thank you!

cc. @gkdn

**Comment by gkdn** [comment]
The issue is the new `@maven` dependency, which complicates the setup of the workspace. It might be possible that we may switch to new bzlmod instead of trying to handle this kind of problems.
